newrelic-agent/src/main/resources/META-INF/excludes

```diff
@@ -88,4 +84,11 @@
 # exclusions for Open Liberty 21+ so transactions start properly
 ^com/ibm/ws/security/jaspi/JaspiServletFilter
 # Websphere specific servlet wrapper class
-^com/ibm/ws/webcontainer/servlet/ServletWrapperImpl
\ No newline at end of file
+^com/ibm/ws/webcontainer/servlet/ServletWrapperImpl
+### All default excludes listed below get ignored when the security agent is enabled
+# exclude java.security and sun.reflect classes from transformation
+^java/security/.*
+# crypto classes can cause class circularity errors if they get too far along in the class transformer
+^javax/crypto/.*
+# saxon xlst classes - see ticket 195949
+^net/sf/saxon.*
```

Comment:
These are the three default exclude rules that are effected by the security agent.